ss/test/storage_bench_test.go

```diff
@@ -0,0 +1,194 @@
+//go:build rocksdbBackend
+// +build rocksdbBackend
+
+package sstest
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/iavl"
+	"github.com/sei-protocol/sei-db/proto"
+	"github.com/sei-protocol/sei-db/ss/pebbledb"
+	"github.com/sei-protocol/sei-db/ss/rocksdb"
+	"github.com/sei-protocol/sei-db/ss/sqlite"
+	"github.com/sei-protocol/sei-db/ss/types"
+)
+
+var (
+	backends = map[string]func(dataDir string) (types.StateStore, error){
```

Comment:
The test doesn't really need to be aware of different backends right? Can we just use the the SS interface?